pkg/gds/emails/templates/review_request.html

```diff
@@ -13,20 +13,19 @@
 <ul>
 	<li>ID: <strong>{{ .VID }}</strong></li>
 	<li>Token: <strong>{{ .Token }}</strong></li>
+  <li>Registered Directory: <strong>{{ .RegisteredDirectory }}</strong></li>
 </ul>
 
 <p>To accept the registration, use the command line tool as follows:</p>
 
-<p><code>$ gds review -i {{ .VID }} -t {{ .Token }} -a</code></p>
+<p><code>$ gds review -i {{ .VID }} -t "{{ .Token }}" -a</code></p>
 
 <p>To reject the registration, provide a rejection message as the last argument:</p>
 
-<p><code>$ gds review -i {{ .VID }} -t {{ .Token }} -R -m ""</code></p>
+<p><code>$ gds review -i {{ .VID }} -t "{{ .Token }}" -R -m ""</code></p>
```

Comment:
This has been bugging me forever; I'm so happy quotes are around the token so I can copy and paste that much easier!